src/plugins/embeddable/public/embeddable_panel/panel_header/use_embeddable_panel_context_menu.tsx

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
Should this just be a component instead of a hook?

Reply:
Probably! Will update that.